src/main/webapp/resources/js/pages/projects/processing/ProcessingPriorities.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  fetchProcessingInformation,
+  updateProcessingPriority,
+} from "../../../apis/projects/settings";
+import { Form, notification, Select } from "antd";
+
+export function ProcessingPriorities({ projectId }) {
+  const [priorities, setPriorities] = React.useState([]);
+  const [priority, setPriority] = React.useState();
+
+  const getProcessingInfo = React.useCallback(async () => {
+    const data = await fetchProcessingInformation(projectId);
+    setPriorities(data.priorities);
+    setPriority(data.priority);
+  }, [projectId]);
+
+  React.useEffect(() => {
+    getProcessingInfo();
+  }, [getProcessingInfo]);
+
+  const update = (value) => {
+    updateProcessingPriority(projectId, value).then((message) => {
+      setPriority(value);
+      notification.success({ message });
```

Comment:
Probably should display an error notification if updating the priority fails for any reason

Reply:
Updated in 7429af4